drivers/rf-transceiver/ad9361/ad9361.c

```diff
@@ -484,41 +484,42 @@ static const uint8_t full_gain_table[RXGAIN_TBLS_END][SIZE_FULL_TABLE][3] = {
 	}
 };
 
-static const int8_t full_gain_table_abs_gain[RXGAIN_TBLS_END][SIZE_FULL_TABLE] = {
+static const int8_t full_gain_table_abs_gain[RXGAIN_TBLS_END][SIZE_FULL_TABLE]
+= {
 	{  /* 800 MHz */
 		-1, -1, -1, 0, 1, 2, 3, 4,
```

Comment:
Why not fix this line too? In  raw mode it still looks bad

Reply:
This was handled by the code style script. I will try to adjust them manually

Reply:
I tried rearranging them in different manners, the indenting is kept the same (first line indented with one tab less) after applying the code style. Either we discard this change and the build will fail, either we keep this change as it is.

Reply:
Is this needed? Coding style is ignored on master.

Reply:
We can ignore it now, but as soon as this file will be changed again, we will face the same build issue at PR.